R/shared_constants.R

```diff
@@ -34,6 +34,7 @@ rx_static_token <- local({
 rx_unescaped_regex <- paste0("(?s)", rex(start, zero_or_more(rx_non_active_char), end))
 rx_static_regex <- paste0("(?s)", rex(start, zero_or_more(rx_static_token), end))
 rx_first_static_token <- paste0("(?s)", rex(start, zero_or_more(rx_non_active_char), rx_static_escape))
+rx_escapable_tokens <- "^${}().*+?|[]\\<>=:;/_-!@#%&,~"
```

Comment:
How did this diff get in here?

Reply:
This is newly linted:

https://github.com/r-lib/lintr/blob/72eeefd87596ea7fa72ee1a8b14bcb82d322c381/R/shared_constants.R#L105-L111

I was originally going to move the comment to the end of the line, so pulled out this constant to keep things within 120. In the end opted to move the comment as shown.

Kept this change because I still think it's better -- abstracts away the logic for readability.